concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+Zalizniak-2020-2591-240	240	corner	CORNER	192	240:—:15 240:—:3 240:—:3 240:→:2 240:→:1 240:—:1	shift437 shift1207 shift1293 shift118 shift1030 shift4297	6	25
+Zalizniak-2020-2591-241	241	comb (of a bird)	COMB OF BIRD	188	241:—:4 241:—:3 241:—:2 241:—:1 241:—:1 241:→:1	shift509 shift4173 shift511 shift510 shift3237 shift512	6	12
+Zalizniak-2020-2591-242	242	can, to be able	BE ABLE	972	242:→:4 242:→:3 242:→:2 242:—:2 242:→:1 242:→:1	shift1773 shift1758 shift1757 shift362 shift2582 shift3492	6	13
+Zalizniak-2020-2591-243	243	calm	CALM	258	243:→:6 243:—:2 243:—:2 243:—:1 243:→:1 243:—:1	shift1144 shift3941 shift2751 shift3101 shift2191 shift1505	6	13
+Zalizniak-2020-2591-244	244	brain (cerebrum)	BRAIN	1583	244:→:10 244:—:10 244:—:3 244:—:1 244:—:1 244:→:1	shift1950 shift1025 shift222 shift4061 shift2222 shift529	6	26
+Zalizniak-2020-2591-245	245	bowels, intestine	INTESTINES	1991	245:↔:6 245:→:3 245:→:3 245:→:2 245:→:1 245:→:1	shift3300 shift1496 shift3562 shift3563 shift230 shift3564	6	16
+Zalizniak-2020-2591-246	246	black cloud	STORM CLOUDS	448	246:—:3 246:→:2 246:—:2 246:→:1 246:—:1 246:—:1	shift2814 shift1440 shift418 shift3379 shift1054 shift1053	6	10
+Zalizniak-2020-2591-247	247	bird	BIRD	937	247:—:4 247:→:2 247:→:2 247:→:2 247:—:1 247:—:1	shift2846 shift2564 shift2776 shift2923 shift4066 shift2812	6	12
+Zalizniak-2020-2591-248	248	arrow	ARROW	977	248:→:7 248:—:4 248:—:1 248:→:1 248:→:1 248:—:1	shift2379 shift2391 shift684 shift210 shift2389 shift657	6	15
+Zalizniak-2020-2591-249	249	air	AIR	27	249:—:12 249:—:8 249:—:4 249:—:3 249:→:2 249:→:1	shift904 shift48 shift683 shift3099 shift2271 shift3155	6	30
+Zalizniak-2020-2591-250	250	<vessel>			250:↔:23 250:↔:13 250:—:5 250:→:2 250:—:1 250:→:1	shift977 shift2647 shift421 shift3273 shift1507 shift3151	6	45
+Zalizniak-2020-2591-251	251	year	YEAR	1226	251:—:3 251:—:1 251:—:1 251:—:1 251:—:1	shift3885 shift579 shift407 shift3887 shift4064	5	7
+Zalizniak-2020-2591-252	252	woman	WOMAN	962	252:→:29 252:→:2 252:—:2 252:↔:1 252:—:1	shift966 shift3109 shift1533 shift562 shift125	5	35
+Zalizniak-2020-2591-253	253	wide, broad	BROAD OR WIDE	2652	253:—:4 253:—:2 253:→:2 253:—:1 253:—:1	shift2415 shift226 shift2543 shift2808 shift2860	5	10
+Zalizniak-2020-2591-254	254	warm	WARM	1232	254:→:7 254:→:3 254:—:2 254:→:1 254:→:1	shift2103 shift160 shift3672 shift4057 shift3250	5	14
+Zalizniak-2020-2591-255	255	untruth	LIE (FALSEHOOD)	2826	255:—:3 255:—:2 255:—:1 255:—:1 255:—:1	shift252 shift1027 shift122 shift2230 shift105	5	8
+Zalizniak-2020-2591-256	256	trunk (body part)	TORSO	2667	256:—:2 256:→:2 256:→:2 256:—:2 256:—:1	shift1550 shift564 shift2875 shift431 shift734	5	9
+Zalizniak-2020-2591-257	257	to work	WORK (ACTIVITY)	1086	257:↔:21 257:↔:18 257:→:7 257:—:2 257:—:1	shift1164 shift1163 shift1165 shift2016 shift3642	5	49
+Zalizniak-2020-2591-258	258	to walk, wander	WANDER	3666	258:—:3 258:→:3 258:→:1 258:—:1 258:→:1	shift49 shift1013 shift212 shift1071 shift3490	5	9
```

Comment:
WALK is already mapped to the gloss _to walk, to go_: Zalizniak-2020-2591-489. I mapped it to WANDER because the second gloss _go_ vs. _walk_ seemed to disambiguate the two concepts.